setup.py

```diff
@@ -39,7 +39,7 @@
       scripts=glob('scripts/*py'),
       setup_requires=['numpy >= 1.9.2'],
       install_requires=[
-          'IPython < 4.0.0',
+          'IPython',
```

Comment:
ipymd hasn't had a commit in months, so I highly suspect the issue still
exists. When we spoke over the on Friday, you indicated you were doing an
explicit conversion from .ipynb to .md which suggests it is broken

On Tue, Dec 22, 2015 at 10:04 AM, J W Debelius <notifications@github.com>
wrote:

> In setup.py
> <https://github.com/biocore/American-Gut/pull/182#discussion_r48279967>:
>
> > @@ -39,7 +39,7 @@
> >        scripts=glob('scripts/*py'),
> >        setup_requires=['numpy >= 1.9.2'],
> >        install_requires=[
> > -          'IPython < 4.0.0',
> > +          'IPython',
>
> @wasade <https://github.com/wasade>, did you get a chance to test this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/American-Gut/pull/182/files#r48279967>.
>